src/components/pages/Auth/Auth.js

```diff
@@ -8,9 +8,21 @@ import './Auth.scss';
 class Auth extends React.Component {
   authenticateUser = (e) => {
     e.preventDefault();
-    authRequests.authenticate().then(() => {
-      this.props.history.push('/home');
-    }).catch(err => console.error('error in authenticating', err));
+    authRequests.authenticate()
+      .then((results) => {
+        userRequests.getUserByUid(results.user.uid)
+          .then((userObject) => {
+            if (!userObject) {
+              const newUserObject = {
+                userName: `${results.user.displayName}`,
+                photo: `${results.user.photoURL}`,
+                uid: `${results.user.uid}`,
+              };
+              userRequests.createUser(newUserObject);
```

Comment:
since this is a promise it should have a .then on it.  This line;
`this.props.history.push('/home');`

should be inside the .then